docs/connector-development/connector-builder-ui/error-handling.md

```diff
@@ -0,0 +1,84 @@
+
+### Wait time from header
+
+The "Wait time from header" backoff strategy allows the connection to wait before retrying a request based on the value specified in the API response.
+
+#### Example
+
+The [Chargebee API](https://apidocs.chargebee.com/docs/api/error-handling) documentation recommends using the `Retry-After` in the response headers to determine when to retry the request.
+
+When running a sync, if the API responds with a 429 error response, the connection will interpret the response and check the `Retry-After` header to see when to attempt the request again.
+
+### Wait until time from header
+
+The "Wait until time from header" backoff strategy allows the connection to wait until a specific time before retrying a request according to the API response.
+
+#### Example
+
+The [Recurly API](https://recurly.com/developers/api/v2021-02-25/index.html#section/Getting-Started/Limits) is an API that defines a header `X-RateLimit-Reset` which specifies when the request rate limit will be reset. 
+
+During a sync, if the connection receives a 429 error response, it will read the `X-RateLimit-Reset` header and pause making further requests to the Recurly API until after the time specified in the header.
+
+## Response filter
+
+A response filter should be used when a connection needs to interpret an API response to decide how the sync should proceed. Common use cases for this feature include ignoring error codes to continue fetching data, retrying requests for specific error codes, and stopping a sync based on the response received from the API.
+
+### Response conditions
+
+The following conditions can be specified on the "Response filter" and are used to determine if attributes of the response match the filter:
+- "If error message matches": The filter will match the provided text against the text body of the API response.
+- "and predicate is fulfilled": This allows for more granular control over how the filter matches by evaluating an interpolation expression against the API response.
+- "and HTTP codes match": The filter will check to see if the HTTP status code of the response is in the provided set of HTTP status codes.
+
+### Then execute action
+
+If a response from the API matches the predicates of the response filter the connection will continue the sync according to the "Then execute action" definition. This is a list of the actions that a connection can take:
+- SUCCESS: The response was successful and the connection will continue fetching the next set of records from the API.
+- RETRY: The response was unsuccessful, but the error is transient and may be successful on subsequent attempts. The request will be retried according to the backoff policy defined on the error handler.
+- IGNORE: The response was unsuccessful, but the error should be ignored and the connection can continue fetching the next set of records from the API.
```

Comment:
What's the difference between success and ignore?

Reply:
good question :)

We should specify we'll extract records from a successful response, but not from an error we ignore

Reply:
added